src/api/routes/company.js

```diff
@@ -33,17 +33,24 @@ export default (app) => {
         authMiddleware.isCompany,
         companyMiddleware.profileNotComplete,
         fileMiddleware.parseSingleFile("logo"),
-        validators.finish,
         fileMiddleware.localSave,
         fileMiddleware.cloudSave,
+        fileMiddleware.parseArrayOfFiles("images", Company.images.max_length, true),
+        fileMiddleware.localSaveArray,
+        fileMiddleware.cloudSaveArray,
+        validators.finish,
         async (req, res, next) => {
 
             try {
                 const companyService = new CompanyService();
-                const { bio, contacts } = req.body;
+                const { bio, contacts, social, locations } = req.body;
                 const logo = req?.file?.url || `${config.webserver_host}/static/${req.file.filename}`;
+                const images = req?.files?.map((file) => file?.url || `${config.webserver_host}/static/${file.filename}`) ?? [];
```

Comment:
Perhaps this should be a function: should we need this logic elsewhere we only need to change it in one place.